### PR TITLE
TASK-7: Queues - connect front and back

### LINKED ISSUE #13: TASK-7: Queues - connect front and back

### PR DESCRIPTION
**Pull request by avdeev1**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -29,10 +29,6 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-data-rest</artifactId>
-        </dependency>
```
Comment (olli2017) [inline pom.xml:35]: why you delete this? Is it garbage?

Comment (avdeev1) [reply pom.xml:35]: duh

File: src/main/java/com/star_trello/darkside/repo/QueueRepo.java [hunk 2]
```diff
@@ -23,4 +23,11 @@
         return getByTitle(title).getTaskList();
     }
 
+    default Queue getQueueInfo(String title) {
+        Queue queue = getByTitle(title);
+        queue.setCreator(null);
```
Comment (olli2017) [inline src/main/java/com/star_trello/darkside/repo/QueueRepo.java:28]: why we need set this to null? It is by default, isn't it?

**Review by sophiekh — APPROVED** [review]

File: src/main/java/com/star_trello/darkside/repo/QueueRepo.java (hunk 2 above)
Comment (avdeev1) [reply src/main/java/com/star_trello/darkside/repo/QueueRepo.java:28]: no, that handler returns creator credentials

Comment (avdeev1) [reply src/main/java/com/star_trello/darkside/repo/QueueRepo.java:28]: but, maybe in future i'll delete this handler, i'm not sure, will it need to me

File: src/main/java/com/star_trello/darkside/service/QueueService.java [hunk 3]
```diff
@@ -55,4 +69,15 @@
         }
         return ResponseEntity.ok().body(queueRepo.getTasksByQueueTitle(title));
     }
+
+    @Transactional
+    public ResponseEntity<?> getQueueByTitle(String token, String title) {
+        if (!userSessionRepo.existsByToken(token)) {
+            return ResponseEntity.badRequest().body("Token is invalid.");
```
Comment (olli2017) [inline src/main/java/com/star_trello/darkside/service/QueueService.java]: strings 32, 76. Why different logic?

**Review by olli2017 — APPROVED** [review]
nice. in future we will remove duplicates! god job bro

**Pull request merged**